### ISSUE TITLE
DIVI-Intensivregister

### ISSUE THREAD

**Issue by nevrome** [nevrome/covid19germany#41]
https://www.intensivregister.de/#/index

Another valuable data source that could be made available with this package.

**Comment by stschiff** [comment]
Yes, that would be super useful for modelling. I might propose something if I find the time...
